Procfile

```diff
@@ -1,2 +1,3 @@
-web: npm run start:prod
+web: docker compose up
+# web: npm run start:prod
```

Comment:
Same question here

Reply:
commented out temporarily while testing heroku, fixed in latest commit.